### PR TITLE
Clize topographic band CLIs

### PR DESCRIPTION
**Pull request by bayliffe**
I simplified the CLIs by removing the checking for existing files in the destination location, as we do not do this with any other CLIs.  As such the behaviour is to create the requested output even if a file exists at the destination path.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

File: lib/improver/cli/generate_topography_bands_weights.py [hunk 1]
```diff
@@ -31,95 +31,16 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Script to run topographic bands weights generation."""
 
-import os
+from improver import cli
+from improver.cli.generate_topography_bands_mask import THRESHOLDS_DICT
```
Comment (arh89) [inline lib/improver/cli/generate_topography_bands_weights.py]: Could we move the import inside the `process` function, please?

File: lib/improver/cli/generate_topography_bands_mask.py [hunk 2]
```diff
@@ -50,72 +43,12 @@
                    'units': 'm'}
```
Comment (arh89) [inline lib/improver/cli/generate_topography_bands_mask.py]: Can this dictionary be moved somewhere else? e.g: in the `generate_ancillaries` module?

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ lib/improver/tests/acceptance/test_generate_topography_bands.py  -1
+ lib/improver/cli/generate_topography_bands_mask.py  -3
+ lib/improver/cli/generate_topography_bands_weights.py  -2
         

Clones removed
==============
+ lib/improver/cli/generate_topography_bands_mask.py  -1
+ lib/improver/cli/generate_topography_bands_weights.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4684489&bid=15666007)

File: lib/improver/cli/generate_topography_bands_weights.py (hunk 1 above)
Comment (bayliffe) [reply lib/improver/cli/generate_topography_bands_weights.py]: Done.

File: lib/improver/cli/generate_topography_bands_mask.py (hunk 2 above)
Comment (bayliffe) [reply lib/improver/cli/generate_topography_bands_mask.py]: Done.

Comment (bayliffe) [reply lib/improver/cli/generate_topography_bands_mask.py]: Done.

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ lib/improver/tests/acceptance/test_generate_topography_bands.py  -1
+ lib/improver/cli/generate_topography_bands_mask.py  -3
+ lib/improver/cli/generate_topography_bands_weights.py  -2
         

Clones removed
==============
+ lib/improver/cli/generate_topography_bands_mask.py  -1
+ lib/improver/cli/generate_topography_bands_weights.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4684489&bid=15666007)

File: lib/improver/cli/generate_topography_bands_mask.py (hunk 2 above)
Comment (arh89) [reply lib/improver/cli/generate_topography_bands_mask.py]: Thanks 👍

**Pull request merged**